index.js

```diff
@@ -244,22 +271,43 @@ module.exports = function (schema, options) {
             this.deletedBy = deletedBy;
         }
 
+        if (schema.path('deletedId')) {
+            this.deletedId = deletedId || deletedBy; // Use deletedId if provided, fall back to deletedBy
+        }
```

Comment:
🤔 I don't think you should fallback to `deletedBy` here. If deletedBy wasn't specified in the code above this then `this.deletedBy` would just be set to undefined. I think this should behave similarly